src/Wrapper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Factory;
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Factory\Definitions\ArrayDefinition;
+use Yiisoft\Factory\Definitions\DefinitionInterface;
+
+class Wrapper implements ContainerInterface
```

Comment:
It needs to be made final and marked as `@internal`. Also a better name would be something like `DelegateToFactoryContainer`.

```suggestion
/**
 * @internal
 */
final class Wrapper implements ContainerInterface
```